packages/ui/__tests__/NumberFieldFormik.test.tsx

```diff
@@ -111,16 +113,17 @@ describe('NumberFieldFormik', () => {
 
     const wrapper = await mountAndCheckA11Y(<TestForm />)
 
-    expect(wrapper.find(Error).prop('error')).toBe(undefined)
+    expect(wrapper.find(Error).prop('error')).toBe('Dark side')
 
     // We need the `async` call here to wait for processing of the asynchronous 'change'
     // eslint-disable-next-line @typescript-eslint/require-await
     await act(async () => {
-      simulateChange(wrapper.find('input'), '56')
+      simulateChange(wrapper.find('input'), 56)
     })
-    wrapper.update()
 
-    // The error is displayed only when the input becomes dirty
-    expect(wrapper.find(Error).prop('error')).toBe('Dark side')
+    expect(formikBag.current?.values).toEqual({
+      name: 56,
+    })
```

Comment:
nit: we are testing the error here. I would not check the values in this test, just the errors